content/en/docs/pipelines/sdk/pipelines-metrics.md

```diff
@@ -23,8 +23,15 @@ experiment in the Kubeflow Pipelines UI.
  
 ## Export the metrics file
 
-To enable metrics, your program must write out a file named 
-`/mlpipeline-metrics.json`. For example:
+To enable metrics, your component must write a JSON file specifying metrics to render.
+The pipeline component must also export a file output artifact with an artifact name of `mlpipeline-metrics`,
+or else the Kubeflow Pipelines UI will not render the visualization. In other words,
+the `.outputs.artifacts` setting for the generated pipeline component should show:
+`- {name: mlpipeline-metrics, path: /mlpipeline-metrics.json}`.
+The JSON filepath does not matter, although `/mlpipeline-metrics.json`
+is used for consistency in the examples below.
+
+Example JSON content:
```

Comment:
LGTM @lucinvitae Thank you. If you have time, can you wrap the text at 80, such as:

```
To enable metrics, your component must write a JSON file specifying metrics to
render. The pipeline component must also export a file output artifact with an
artifact name of `mlpipeline-metrics`, or else the Kubeflow Pipelines UI will
not render the visualization. In other words, the `.outputs.artifacts` setting
for the generated pipeline component should show: `- {name: mlpipeline-metrics,
path: /mlpipeline-metrics.json}`. The JSON filepath does not matter, although
`/mlpipeline-metrics.json` is used for consistency in the examples below.

Example JSON content:
```